### ISSUE TITLE
Bad Proxy error while generating accounts 

### ISSUE THREAD

**Issue by CodeOne45** [Vinyzu/MjolnirAiO#6]
Hello, I'm facing the following error when a wanted to generate accounts : 
![image](https://user-images.githubusercontent.com/57890543/188383249-d3ffdbcc-792b-417f-ac39-fba4f4d42d00.png)
 
Do you have any idea why I'm facing this error?

**Comment by Vinyzu** [comment]
Are you using proxies?

**Comment by CodeOne45** [comment]
yes I'm using a proxy server and also when I use a proxy I got this error: 
![image](https://user-images.githubusercontent.com/57890543/189604534-48a1fd5a-0ae3-468c-b704-e3e0ebc853ca.png)

So, the account creation doesn't work

**Comment by Vinyzu** [comment]
Its probably either cause your proxy providrr banned spotify, spotify banned your proxy provider or an isp issue.

**Comment by janya171** [comment]
Please check this s**potify view and follow bot**; no browser type bro :) I hope it help
https://youtu.be/3BGeeGlYTsU
